### ISSUE TITLE
Nuevas caracteristicas

### ISSUE THREAD

**Issue by jorgepj99** [jgons18/jocpython#5]
1.-Podriamos hacer una nueva sala taller en la que se podrian crear cosas
2.-Hacer que los items a recoger sean piezas de construccion
3.-Empezar el juego con la llave rota y que tenga que soldarse o fabricarse una nueva
4.-Poner varios items en las diferentes salas

**Comment by jgons18** [comment]
Me parece bien el punto 3.
Poner algo tipo "Tienes 1/4 de la llave" y así sucesivamente,entonces cuando tenga todas las piezas, ir al taller a soldarlas

**Comment by jorgepj99** [comment]
Entonces tambien te parece bien el punto 1???

**Comment by jgons18** [comment]
Sí, igual en vez de poner una sala entera para el taller, utilizar una parte de la planta del sótano.

**Comment by jorgepj99** [comment]
Si esa era la idea

**Comment by jorgepj99** [comment]
Voy ha hacerlo
